src/ffi/pkgconfig.nit

```diff
@@ -80,7 +80,7 @@ class PkgconfigPhase
 			proc_exist.wait
 			status = proc_exist.status
 			if status == 1 then
-				modelbuilder.error(nat, "Error: package `{pkg}` unknown by `pkg-config`, make sure the development package is be installed.")
+				modelbuilder.error(nat, "Error: dev package for `{pkg}` unknown by `pkg-config`, install it with `apt-get`, `brew` or similar.")
```

Comment:
May I propose``"Error: dev package for `{pkg}` not found by `pkg-config`. Please install it with `apt-get`, `brew` or similar."``?